genie-web/src/main/java/com/netflix/genie/web/properties/AgentFileStreamProperties.java

```diff
@@ -0,0 +1,74 @@
+/*
+ *
+ *  Copyright 2020 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+package com.netflix.genie.web.properties;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.validation.annotation.Validated;
+
+import java.time.Duration;
+
+/**
+ * Properties related to {@link com.netflix.genie.web.agent.services.AgentFileStreamService}.
+ *
+ * @author mprimi
+ * @since 4.0.0
+ */
+@ConfigurationProperties(prefix = AgentFileStreamProperties.PROPERTY_PREFIX)
+@Getter
+@Setter
+@Validated
+public class AgentFileStreamProperties {
+
+    /**
+     * Prefix for all properties related to the agent file transfer service.
+     */
+    public static final String PROPERTY_PREFIX = "genie.agent.filestream";
```

Comment:
Nit: should it be `file-stream`?

Reply:
We're not super consistent. Some have dashes, other don't, e.g.: `genie.s3filetransfer`

Reply:
we could at least be consistent about any new ones but if you'd rather sweep through and standardize them all later then that's fine